src/Dfe.PlanTech.AzureFunctions/Mappings/ComponentDropDownMapper.cs

```diff
@@ -6,8 +6,5 @@ namespace Dfe.PlanTech.AzureFunctions.Mappings;
 
 public class ComponentDropDownMapper(EntityRetriever retriever, EntityUpdater updater, ILogger<JsonToDbMapper<ComponentDropDownDbEntity>> logger, JsonSerializerOptions jsonSerialiserOptions) : JsonToDbMapper<ComponentDropDownDbEntity>(retriever, updater, logger, jsonSerialiserOptions)
```

Comment:
what's this class used for then?

Reply:
It maps the `ComponentDropDown` content type from Contentful to the `ComponentDropDownDbEntity` that it needs to be saved as. 

It appears empty as there's no extra mapping it requires that the base class doesn't handle, unlike a lot of the others. There's a few "empty" ones: Button, ComponentDropDown, Header, InsetText, Title. Any that don't reference another entity, basically.